### PR TITLE
[FIX] Updated EditLegacy Component

### PR DESCRIPTION
**Pull request by blackdevelopa**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**
This fix creates and uses an updated EditLegacy component for legacy transactions in the Send Flow. The default EditLegacy component is reverted to prevent breaking the app for other legacy transactions (approve, approval and swaps).

_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions,_
_1. What is the reason for the change?_
_2. What is the improvement/solution?_

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Review by sethkfman — COMMENTED** [review]
Could we just replace the old Gas Fee Legacy component and remove the unused code?

**Review by sethkfman — APPROVED** [review]
LGTM

File: app/components/UI/EditGasFeeLegacy/index.js [hunk 1]
```diff
@@ -25,7 +25,6 @@ import FadeAnimationView from '../FadeAnimationView';
 import AnalyticsV2 from '../../../util/analyticsV2';
 import AppConstants from '../../../core/AppConstants';
 import { useTheme } from '../../../util/theme';
-import { useGasTransaction } from '../../../core/GasPolling/GasPolling';
 
 const GAS_LIMIT_INCREMENT = new BigNumber(1000);
 const GAS_PRICE_INCREMENT = new BigNumber(1);
```
[the remark this answers is not in the document]
Comment (blackdevelopa) [reply app/components/UI/EditGasFeeLegacy/index.js:30]: Yep. Did that [here](https://github.com/MetaMask/metamask-mobile/pull/5116/commits/c81ae463c915861f2c0180876ed959e1a6c852ff). Thanks

**Review by tommasini — APPROVED** [review]
Great work!! Just a question about the typing, would be better type the useState and the arguments of functions on this new component `EditGasFeeLegacyUpdate`?

**Comment by chrisleewilcox** [comment]
I am not able to cancel a Send transaction on goerli network.  In the recording you can see I canceled before transaction was successful...
https://recordit.co/mdXbXqRygH

**Comment by chrisleewilcox** [comment]
LGTM.

**Pull request merged**